base/tests/test_forms.py

```diff
@@ -272,6 +289,7 @@ async def test_submit_to_healthconnect(self):
                 "city_location_coords": "+1.2-3.4",
                 "location_coords": "+3.4-1.2",
                 "location": "Cape Town, South Africa",
+                "research_consent": "no",
```

Comment:
I think the API is expecting boolean or None

Reply:
This applies to all payload tests below

Reply:
We mapping it on the payload, then i will send True or False

Reply:
I see, nice!